project/newrelic_webmon/newrelic.ini

```diff
@@ -0,0 +1,195 @@
+# ---------------------------------------------------------------------------
```

Comment:
In recent projects I've found myself skipping this and just configuring everything via the environment variables.